### DISCUSSION TITLE
第5节课笔记(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#306] [笔记]
Here

**Comment by al2h36o30s3** [comment]
https://zhuanlan.zhihu.com/p/677777804

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135588377

**Comment by ltstar** [comment]
[课程5-LMDeploy.md](https://github.com/InternLM/tutorial/files/13958339/5-LMDeploy.md)

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/a844c9e5757c449fb11eb0b4aa8c313b?pvs=4

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135659700

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec5/

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135708125?spm=1001.2014.3001.5501

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/YIL8deDgWoab1WxlDgkcfSednwb
